### ISSUE TITLE
Remove romsmania.cc from the list

### ISSUE THREAD

**Issue by pighyboy** [Igglybuff/awesome-piracy#653]
It's been proven on multiple subreddits it's full of viruses, just like it's sister sites romsmania.com and roms-mania.com
Please remove

**Comment by Shakil-Shahadat** [comment]
Can you show some example subreddit?

**Comment by pighyboy** [comment]
@Shakil-Shahadat  
Sorry for not answering, i don't really use GitHub, but you should totally remove romsmania.cc from the list as it does include nonethless, malware. You could probably reference to the r/piracy abandoned megathread and make some changes and additions, or maybe i could become a "curator" of this library, i know a lot about piracy

**Comment by Shakil-Shahadat** [comment]
@pighyboy This is not my list. I used to contribute here but this repository hasn't been updated for some time. You are welcome to contribute to my maintained fork here https://github.com/Shakil-Shahadat/awesome-piracy. I have already added caution [here](https://github.com/Shakil-Shahadat/awesome-piracy#roms). Some said, if someone stays careful, romsmania can be a good source of roms, that's why I'm reluctant to remove it from my fork. I think I should deprioritize it and add additional caution just to inform better.
